src/zenml/steps/step_context.py

```diff
@@ -130,6 +132,14 @@ def metadata_store(self) -> "BaseMetadataStore":
         """
         return self._metadata_store
 
+    @property
+    def feature_store(self) -> Optional["BaseFeatureStore"]:
```

Comment:
I would not put this as a direct property here. I would rather expose the entire active stack, i.e.

```python
@property
    def stack(self) -> Optional["BaseStack"]:
            return Repository().active_stack
```

That way we do not have to worry about optional stack components like feature_stores directly here